src/graphql/data/util.tsx

```diff
@@ -133,10 +133,17 @@ export const CHAIN_NAME_TO_CHAIN_ID: { [key in Chain]: SupportedChainId } = {
 }
 
 export function fromGraphQLChain(chain: Chain): SupportedChainId {
-  return CHAIN_NAME_TO_CHAIN_ID[chain]
+  return CHAIN_NAME_TO_CHAIN_ID[chain] ?? SupportedChainId.MAINNET
 }
 
-export const BACKEND_CHAIN_NAMES: Chain[] = [Chain.Ethereum, Chain.Polygon, Chain.Optimism, Chain.Arbitrum, Chain.Celo]
+export const GQL_SUPPORTED_CHAINS: Chain[] = [
```

Comment:
so my comment was more trying to make this `SupportedChainId[]` (or a separate `SupportedGqlChain` type that is just the gql name, instead of using the GraphQL type `Chain`, that way we can filter the data returned by gql and then they can be either SupportedChainId or SupportedGqlChain and you wouldn't have to loosen the type for `CHAIN_NAME_TO_CHAIN_ID`

lmk this makes sense - happy to draft a PR to show an example

Reply:
Are you thinking something like?:
```
const UX_SUPPORTED_CHAINS = [
  Chain.Ethereum,
  Chain.EthereumGoerli,
  Chain.EthereumSepolia,
  Chain.Polygon,
  Chain.Celo,
  Chain.Optimism,
  Chain.Arbitrum,
  Chain.UnknownChain,
  Chain.Bnb,
] as const
type UxSupportedChainsType = typeof UX_SUPPORTED_CHAINS[number]

export const CHAIN_NAME_TO_CHAIN_ID: { [key in UxSupportedChainsType]: SupportedChainId } = { ... }

Reply:
Otherwise would appreciate a snippet to know more of what you're thinking